docs/source/en/api/pipelines/diffedit.mdx

```diff
@@ -357,4 +343,4 @@ images[0].save("edited_image.png")
     - all
     - generate_mask
     - invert
-	- __call__
```

Comment:
Why are we deleting the non-call methods?

Reply:
Should be back in now, not sure why it got removed in the first place.